### ISSUE TITLE
need pretrained weights file on something other than baidu

### ISSUE THREAD

**Issue by nsharanarthi-dori** [youngnuaa/iShapeInst#3]
Hello,

People outside china cannot use baidu to download the pretrained weights, i do not posses a chinese phone number to register an account. please share it using google drive or any other cloud provider who has worldwide access. 

Appreciate it!

**Comment by youngnuaa** [comment]
Thank you for your attention to our work. We have uploaded all the models to Google Drive. Please check and download them. https://drive.google.com/drive/folders/1c6AX6wLb5GnNi0FsvQ0KZtkrpx00UPy-?usp=sharing
&nbsp;





&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2025年8月8日(星期五) 晚上7:50
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [youngnuaa/iShapeInst] need pretrained weights file on something other than baidu (Issue #3)












nssharanarthi created an issue (youngnuaa/iShapeInst#3)
 
Hello,
 
People outside china cannot use baidu to download the pretrained weights, i do not posses a chinese phone number to register an account. please share it using google drive or any other cloud provider who has worldwide access.
 
Appreciate it!

**Comment by nsharanarthi-dori** [comment]
Really appreciate the quick response! i assumed you wouldn't check i was happy to see you reply! :)

The issue right now is the Gdrive contains 2 types of files,
1. Binary -> antenna, log, hanger
2. Zip -> branch, fence

but all of these have extension baiduyun.p.downloading, which is assume it hasn't finished downloading completely. 
I renamed it and did torch.load but it threw error. Can you please share the complete file .pt extension after 100% download? 

Thank you so much in advance, excited to see the results! If the results are good, ill fine tune on my custom dataset and publish results.

P.S : What was the loss when the model converged? How many epocs was it trained on do you have the logs by any chance? I'm asking because i tried to train on hanger from scratch it wasnt drawing proper mask maybe it was underfit.

**Comment by nsharanarthi-dori** [comment]
@youngnuaa  Hello, can you please re share the complete file?

**Comment by youngnuaa** [comment]
I will check and upload a new one, I hope it helps you. https://drive.google.com/drive/folders/1c6AX6wLb5GnNi0FsvQ0KZtkrpx00UPy-?usp=sharing
&nbsp;





&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2025年8月13日(星期三) 下午5:21
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [youngnuaa/iShapeInst] need pretrained weights file on something other than baidu (Issue #3)












nssharanarthi left a comment (youngnuaa/iShapeInst#3)
 
@youngnuaa  Hello, can you please re share the complete file?
